.travis.yml

```diff
@@ -1,19 +0,0 @@
-language: csharp
-
-os:
- - linux
- - osx
-
-dist: bionic
-osx_image: xcode9.1
-
-mono: none
-dotnet: 2.1.300
-
-before_install:
-  - cd SimplePolicy.UnitTests
```

Comment:
We should have a travis, just for compilation, and in the future make test for plugins too, but i think that we need it

Reply:
I think it is not needed it there are no tests.
Sometime ago I did a template for each plugin and it was rejected.

Reply:
But it looks strange to have the yellow simbol of travis forever. Maybe we can leave the file as you suggested.

Reply:
Let's leave the file.

Reply:
Re-added without scripts

Reply:
We can add `dotnet restore` to make it build automatically.

Reply:
Done

Reply:
Travis failed.

You can revert the changes to the `.travis.yml` and remove the line `dotnet test` only.

Reply:
It is ok, we need to remove the `before_install` as well.
I think that I forgot to clean the .sln

Reply:
You should clean `SimplePolicy.UnitTests` from .sln too. Why don't you use Visual Studio?

Reply:
AEHuaheuahea, until nowadays using `gedit` or editing here online.
I need to configure visual studio code. I will do that as soon as possible.